src/pages/docs/infrastructure/deployment-targets/tentacle/troubleshooting-tentacles.md

```diff
@@ -1,268 +0,0 @@
----
```

Comment:
We usually keep the markdown file, but turn it into a redirect to an alternative page.

See: https://github.com/OctopusDeploy/docs?tab=readme-ov-file#redirects